### ISSUE TITLE
best way to theme individual webcomics on one install?

### ISSUE THREAD

**Issue by glancereviver** [mgsisk/webcomic#224]
I am designing a site that has a landing site/portfolio, and two webcomics.  The way that the owner previously had set up their site was with three separate installations of wordpress, which I thought was unnecessary, since Webcomic can run multiple webcomics.

So right now I have one wordpress install with two webcomics on it, the theme is Inkblot.  I want to give the two webcomics different themes than the main page, but still based on inkblot.

First I tried making inkblot some child themes.  The child themes work fine if I apply them to the website as a whole by setting them as the active theme in wordpress options, so I know that they are configured correctly.  But in the individual webcomic settings' dropdown menu, when I selected the child theme, it completely ignored the child theme, and only pulls from the master inkblot theme.  

So then I thought to try to clone and rename Inkblot so that the individual webcomics settings recognizes it as a completely different theme all together, but just changing the instances of theme name in the style.css doesn't seem to be enough--it still seems to be pulling most of the styles from Inkblot.

So, to sum up...
Is there a way to get the webcomics to recognize child themes?
Otherwise, is there something in another file that I need to do to successfully clone inkblot?
or, should I have just done several wordpress installs to start with?

**Comment by Mihari** [comment]
Alright, if I'm understanding this correctly, you have one install, multiple webcomics, but you want different themes per webcomic collection.

Well, webcomic does support this feature, however, as of WordPress 4.0 and up, the inkblot theme is quite broken as far as theme customization goes. That's only one of the problems right now. You should be able to duplicate inkblot, rename the folder, edit the style.css to give you a different theme... at least as far as WordPress is concerned. Would you mind sharing your child themes comment header?

**Comment by mgsisk** [comment]
Apologies for the trouble, @glancereviver. Selecting a different theme from the collection settings page for a webcomic should force it to use that theme, but there may be issues with newer versions of WordPress. I'll look into this.

**Comment by mgsisk** [comment]
It looks like the may be at least partly caused by Inkblot's dynamic stylesheet. Still not sure about a reliable workaround (besides cloning the entire Inkblot theme and setting it up as a separate theme). I may have to rework Inkblot's dynamic stylesheet into a more traditional stylesheet with inline styles for theme modifications…

**Comment by mgsisk** [comment]
A combination of 518efbcd96b8b424e05d0e4b2efa10c5aceb6441 and mgsisk/inkblot@06e098ecd5142618335521712cd096f64c3b3440 corrects this issue: Inkblot child themes should now work properly when selected as the active theme for a collection.
